types/lodash-fp/lodash-fp.utils.errors.ts

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import _ from 'lodash/fp'
+
+import { SafeCurry2, SafeCurry3, SafeCurry4 } from '../../src/libs/type-utils/lodash-fp-types'
+
+
+// Curry2 ////////////////////////////////////////////////////////////////
+
+const fnWith2ArgsFn1 = (_x: number, _y: string): string => {
+  return 'something interesting'
+}
+const fnWith2ArgsFn2 = (_x: number[], _y: string) : string => {
+  return 'something interesting'
+}
+const badFn = (_arg1: number): string => {
```

Comment:
What makes this function "bad"? Or does it only appear in examples of incorrect usage?

Reply:
only used in ...errors.ts to force incorrect usage checks

Reply:
added comment